airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -119,27 +122,52 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
     }
   }
 
+  protected void updateSchemaFocus(JsonNode config) {
+    if (config.has("schemaFocus")) {
```

Comment:
does this configuration need to be added to the spec for each database then? i'm curious how you were thinking of exposing this in the ui. are you just thinking of a boolean or is the user entering the schemas they care about?

Reply:
It could just be a boolean in the UI yes.

Or it can be non-visible in the UI and not in the spec.json at all... 

Then all source would list tables from all schemas since you can't specify a specific schemas in the jdbc sources as of now...

Reply:
sorry, I wasn't clear. Which one of these things were you planning to do?

Reply:
let's remove the schemaFocus option so there's nothing to change either in the spec.json or UI